src/CMakeLists.txt

```diff
@@ -252,6 +252,7 @@ register_migraphx_ops(
     undefined
     unique
     unknown
+    unpack_int4
```

Comment:
Please add a URL in comments, if this operator is based on some external specification. Thanks.

Reply:
This is not based on any external specification.